SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorCloudLauncherSettings.cpp

```diff
@@ -62,6 +62,13 @@ bool USpatialGDKEditorCloudLauncherSettings::IsDeploymentNameValid(const FString
 	return RegMatcher.FindNext();
 }
 
+bool USpatialGDKEditorCloudLauncherSettings::IsRegionCodeValid(const ERegionCode::Type RegionCode)
```

Comment:
```suggestion
bool USpatialGDKEditorCloudLauncherSettings::IsRegionCodeValid(const ERegionCode::Type RegionCode) const
```

Reply:
Reverted the const since the method is static.